build.gradle

```diff
@@ -27,6 +30,11 @@ repositories {
 dependencies {
     implementation('org.springframework.boot:spring-boot-starter-data-jpa')
     implementation('org.springframework.boot:spring-boot-starter-web')
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
junit powinien być w wersji 5.x.x jeśli dodałeś `useJUnitPlatform`

Reply:
Poprawiłem na junita5, wymagało to kilku zmian w testach (np. RunWith zmienione na ExtendWith)